### DISCUSSION TITLE
4 Semester 9 Woche

### DISCUSSION THREAD

**Discussion by davidlzt** [MelonigeMelone/xCalendar#40] [Blogkanal]
Heyho,
dieso Woche haben wir uns um eine Retrospektive gekümmert. Dabei war der Ablauf des ganzen Projektes der Mittelpunkt. Wir haben reflektiert, was schiefgelaufen ist und daraus Aktionen beschlossen.
![Screenshot 2024-06-12 155120](https://github.com/MelonigeMelone/xCalendar/assets/113108281/3ab338cd-2056-4791-9bef-0493c33e4e74)
Bei dieser Retro waren alle Teammitglieder anwesend und haben dazu beigetragen.
Bis zum nächsten mal :D

**Comment by MikaK21** [comment]
Hallo zusammen,
ich finde ihr habt gute Punkte gefunden. Zudem habt ihr euch auch direkt Gedanken gemacht, wie ihr speziell das was nicht so gut lief beim nächsten Mal verbessern kann. Finde ich sehr gut!
